plugin-server/src/utils/db/db.ts

```diff
@@ -412,6 +412,34 @@ export class DB {
     }
 
     // Person
+    REDIS_PERSON_ID_PREFIX = 'person_id'
+    REDIS_PERSON_CREATED_AT_PREFIX = 'person_create_at'
+    REDIS_PERSON_PROPERTIES_PREFIX = 'person_props'
+    REDIS_PERSON_INFO_TTL = 1000
```

Comment:
let's make this an env var

Reply:
also - any reason for this particular number? Given it's not a round number in minutes

Reply:
nope, just used a random number, any suggestions for the default